src/libAtomVM/sys.h

```diff
@@ -88,11 +88,11 @@ term sys_get_info(Context *ctx, term key);
 
 void sys_init_platform(GlobalContext *global);
 
-void sys_start_millis_timer();
+void sys_start_millis_timer(void);
 
-void sys_stop_millis_timer();
+void sys_stop_millis_timer(void);
 
-uint32_t sys_millis();
+uint32_t sys_millis(void);
```

Comment:
This looks very weird to me, `void` for functions with no parameters is not needed anymore at least since C99.

Reply:
I was a bit surprised myself, but arm-none-eabi-gcc complains that these functions are undefined without the void.